pubsublite-spark-sql-streaming/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Why do you need this? If you need the inline mock maker (I can't see why you do?) can you change over all our mockito targets to mockito-inline instead?

Reply:
Yea I need it to mock some final methods (in later PRs), let me pull this out and introduce along with the tests.